app/assets/stylesheets/announcements.scss

```diff
@@ -2,12 +2,30 @@
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
 
-h1 {
-  font-size: 2em;
+.sidebar-container {
```

Comment:
Oh omg the h1 is in this. Okay maybe move back to the relevant file? Haha sorry :(

Reply:
Eh? But the sidebar-container class has always been in announcements.scss because it is part of the announcement sidebar at the LHS of homepage.